test/SolidityExpressionCompiler.cpp

```diff
@@ -79,7 +80,9 @@ Declaration const& resolveDeclaration(vector<string> const& _namespacedName,
 	// bracers are required, cause msvc couldnt handle this macro in for statement
 	for (string const& namePart: _namespacedName)
 	{
-		BOOST_REQUIRE(declaration = _resolver.resolveName(namePart, declaration));
+		auto declarations = _resolver.resolveName(namePart, declaration);
+		BOOST_REQUIRE(!declarations.empty());
+		BOOST_REQUIRE(declaration = *declarations.begin());
```

Comment:
I'm using a `std::set` here, so no `.front()` method. It's a bit awkward to use `.begin()` and the dereference it.